Sources/GeneralIOs/Commands/Font.swift

```diff
@@ -0,0 +1,287 @@
+        case notFoundTemplate
+        case somethingGoingWrong(String, String)
+
+        public var errorDescription: String? {
+            switch self {
+            case let .notFoundFonts(path):
+                return red("Is not found fonts in path \(path). Please check correctly path.")
+            case let .notFoundTarget(targetName):
+                return red("Is not found target - \(targetName)")
+            case let .notFoundInfoPlist(path):
+                return red("Is not found info plist by path \(path)")
+            case .invalidData:
+                return red("Data is not valid. Please check correctly Info.plist")
+            case .notFoundTemplate:
+                return red(Constant.notFoundTemplate)
+            case .somethingGoingWrong(let title, let subtitle):
+                return red("""
+                          Something going wrong...
+                          I try it \(title) ...
+                          But I can't perform because \(subtitle).
+                          Please check and try again.
+                          """)
+            }
+        }
+    }
+
+    typealias Dependencies = HasFileHelper &
+                             HasProjectServiceFactory &
+                             HasSpecFactory
+    private lazy var fileHelper = dependencies.fileHelper
+    public static let configuration: CommandConfiguration = .init(abstract: "Add fonts in project.")
+    private lazy var projectService: ProjectService = {
+        let service = dependencies.projectServiceFactory.makeProjectService(path: "./")
+        try? service.createProject(projectName: "\(unwrappedProjectName).xcodeproj")
+        return service
+    }()
+
+    private lazy var specFactory: SpecFactory = dependencies.specFactory
+
+    private lazy var generalSpec: GeneralSpec? = {
+        let pathURL = URL(fileURLWithPath: directoryPath, isDirectory: true)
+        let specURL = URL(fileURLWithPath: Constants.generalSpecName, relativeTo: pathURL)
+        return try? specFactory.makeSpec(url: specURL)
+    }()
+
+    private lazy var unwrappedProjectName: String = {
+        return projectName ?? ask("Please enter project name") ?? ""
+    }()
+
+    private lazy var projectName: String? = {
+        return (try? fileHelper.contentsOfDirectory(at: "./").first { file in
+            file.url.pathExtension == "xcodeproj"
+        })?.url.deletingPathExtension().lastPathComponent
+    }()
+
+    private var dependencies: Dependencies {
+        Services
+    }
+
+    // MARK: - Parameters
+
+    @Option(name: .shortAndLong, help: "Specify a path to folder fonts.")
+    var path: String
+
+    @Option(name: .long, help: "The target to which add files.", completion: .targets)
+    var target: String?
+
+    @Option(name: .shortAndLong, completion: .directory, help: "The path for the project.")
+    var directoryPath: String = FileManager.default.currentDirectoryPath
+
+    // MARK: - Lifecycle
+
+    public func run() throws {
```

Comment:
this function contains too much logic. What about place it in specific service and decompose to separate functions?